ddtrace/span.py

```diff
@@ -223,10 +223,8 @@ def set_traceback(self, limit=20):
         if (exc_type and exc_val and exc_tb):
             self.set_exc_info(exc_type, exc_val, exc_tb)
         else:
-            tb = ''.join(traceback.format_stack(limit=limit))
-            self.set_tag(PYTHON_STACK, tb)
-            # FIXME[gabin] Delete this line when frontend will display "python.stack" tag correctly
-            self.set_tag(errors.ERROR_STACK, tb)
+            tb = ''.join(traceback.format_stack(limit=limit + 2)[:-2])
```

Comment:
looking at that:
```
''.join(traceback.format_stack(limit=limit + 1)[:-1])
```
it seems more reliable since the traceback must include where user code launched the `s.set_traceback()` (only our code must be out but not when users call our code).

With `-2` we have as last line:
```
  File "/Users/emanuele/.virtualenvs/py-tracer3.5/lib/python3.5/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
```

With `-1` we have:
```
  File "/Users/emanuele/workspaces/datadog/dd-trace-py/tests/test_span.py", line 146, in test_traceback_without_error
    s.set_traceback()
```

I think it's better to use `-1` instead of `-2` here.